### PR TITLE
test: automation tests for Transactions + Withdrawal API

### PR DESCRIPTION
**Pull request by olehbairak**
# What ❔

_Increase automation coverage with next test cases:_

Transactions: -> DONE ✅

https://allure.matterlabs.dev/project/11/test-cases/1509 ✅

https://allure.matterlabs.dev/project/11/test-cases/1452 ✅

Withdrawal API: -DONE ✅

https://allure.matterlabs.dev/project/11/test-cases/1459 ✅

https://allure.matterlabs.dev/project/11/test-cases/1460 ✅ 

https://allure.matterlabs.dev/project/11/test-cases/1461 ✅

## Why ❔

Extend automation test coverage with new areas of coverage.

## Checklist

- [ ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.

### CODE REVIEW

File: packages/integration-tests/src/entities.ts [hunk 1]
```diff
@@ -49,6 +49,8 @@ export enum Token {
   ETHER_PULL_Address = "0x0000000000000000000000000000000000008001",
   ETHER_Address = "0x0000000000000000000000000000000000000000",
   ETHER_ERC20_Address = "0x000000000000000000000000000000000000800A",
+  tokenTypeERC20 = "ERC20",
+  tokenTypeETH = "ETH",
```
Comment (pcheremu) [inline packages/integration-tests/src/entities.ts:53]: that is good that you have decided to move it separately, but then you have to align all tests:
image
I would also suggest to simplify the name: 
```
ERC20_Name = "ERC20",
ETH_Name = "ETH",
```

File: packages/integration-tests/tests/api/transactions.test.ts [hunk 2]
```diff
@@ -86,25 +97,33 @@ describe("Transactions", () => {
         apiRoute = `/transactions/${txHash}/transfers`;
         response = await helper.performGETrequest(apiRoute);
 
+        console.log(response.body);
```
Comment (pcheremu) [inline packages/integration-tests/tests/api/transactions.test.ts:100]: do we really need it?

Comment (olehbairak) [reply packages/integration-tests/tests/api/transactions.test.ts:100]: fixed

**Comment by olehbairak** [comment]
@pcheremu closing this PR due to the fact, that these fixes were already present in integration-test-signed branch.
After rebase - no changes left.

**Pull request closed without merging**